packages/p2p-v2/src/pages/buy-sell/screens/BuySellTableRow/BuySellTableRow.tsx

```diff
@@ -0,0 +1,137 @@
+
+    const { completed_orders_count, id, is_online, name, rating_average, rating_count } = advertiser_details || {};
+
+    const { displayEffectiveRate } = generateEffectiveRate({
+        exchange_rate: exchangeRate,
+        local_currency,
+        market_rate: effective_rate,
+        price: price_display,
+        rate,
+        rate_type,
+    });
+    const hasRating = !!rating_average && !!rating_count;
+    const isBuyAdvert = counterparty_type === BUY_SELL.BUY;
+    const isMyAdvert = data?.id === id;
+    const ratingAverageDecimal = rating_average ? Number(rating_average).toFixed(1) : null;
+
+    return (
+        <div className='p2p-v2-buy-sell-table-row'>
+            <Container>
+                <div className='flex flex-row gap-4 items-center'>
+                    <UserAvatar isOnline={is_online} nickname={name || ''} showOnlineStatus size={25} textSize='xs' />
+                    <div className='flex flex-col'>
+                        <div className='flex flex-row items-center gap-2'>
+                            <Text
+                                className={clsx({ 'mb-[-1rem]': hasRating })}
+                                size='sm'
+                                weight={isMobile ? 'bold' : 400}
+                            >
+                                {name}
+                            </Text>
+                            <Badge tradeCount={completed_orders_count} />
+                        </div>
+                        <div className='flex flex-row items-center'>
+                            {hasRating ? (
+                                <>
+                                    <Text className='lg:mr-0 mr-[-1.2rem]' color='less-prominent' size='xs'>
+                                        {ratingAverageDecimal}
+                                    </Text>
+                                    <StarRating
+                                        isReadonly
+                                        ratingValue={Number(ratingAverageDecimal)}
+                                        starsScale={isMobile ? 0.7 : 0.9}
+                                    />
+                                    <Text className='lg:ml-1 ml-[-1rem]' color='less-prominent' size='xs'>
+                                        ({rating_count})
+                                    </Text>
+                                </>
+                            ) : (
+                                <Text color='less-prominent' size='xs'>
+                                    Not rated yet
+                                </Text>
+                            )}
+                        </div>
+                    </div>
+                </div>
+                <Container {...(isMobile && { className: 'flex flex-col ml-14 mt-3' })}>
+                    {isMobile && <Text size='2xs'>Rate (1 USD)</Text>}
+                    <Container {...(isMobile && { className: 'flex flex-col-reverse mb-7' })}>
```

Comment:
Same here. I suggest that since `className` is the only prop here, you shouldn't use object `destructuring` 
the approach below would be better
```suggestion
                    <Container className={isMobile ? 'flex flex-col ml-14 mt-3' : ''}>
```
but if you insist, then this would be the code, though less readable than the above.

`<Container {...(isMobile ? { className: 'flex flex-col ml-14 mt-3' } : {})} />`

Reply:
hmmm I think its fine to leave it as before, as I had already replied to your previous comment, it won't spread the false response, it'll simply return nothing for JSX

Reply:
Sure thing @ameerul-deriv